Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/action/ResourceCommonActionsContributor.java

```diff
@@ -92,7 +92,16 @@ public void registerActions(AzureActionManager am) {
         am.registerAction(DEPLOY, new Action<>(deployView));
 
         final ActionView.Builder createView = new ActionView.Builder("Create", "/icons/action/create.svg")
-                .title(s -> Optional.ofNullable(s).map(r -> title("common|resource.create", ((IAzureResource<?>) r).name())).orElse(null));
+                .title(s -> Optional.ofNullable(s).map(r -> {
+                    String name = r.getClass().getSimpleName();
+                    if (r instanceof IAzureResource) {
+                        name = ((IAzureResource<?>) r).name();
+                    } else if (r instanceof AzureService) {
+                        name = ((AzureService) r).name();
+                    }
+                    return title("common|resource.create", name);
+                }).orElse(null))
+                .enabled(s -> s instanceof IAzureResource || s instanceof AzureService);
```

Comment:
Can actions be accessed in IntelliJ action palette for service without create action

Reply:
yes, this is a problem but not serious, I would fix this problem in later PRs. 
**Actions should be disabled if there are not suitable action handler.**